### PR TITLE
fix(providers): FIRE_HISTORICAL on wekeo_ecmwf

### PR DESCRIPTION
**Pull request by amarandon**
Bring back `FIRE_HISTORICAL` via `wekeo_ecmwf`, now with correct default values and start date.

### CODE REVIEW

File: eodag/resources/providers.yml [hunk 1]
```diff
@@ -4812,6 +4824,34 @@
...
         defaultGeometry: 'POLYGON((180 -90, 180 90, -180 90, -180 -90, 180 -90))'
         orderLink: 'https://gateway.prod.wekeo2.eu/hda-broker/api/v1/dataaccess/download?{{"location": "{downloadLink}","product_id":"{id}", "dataset_id": "EO:ECMWF:DAT:INSITU_GLACIERS_EXTENT"}}'
+    FIRE_HISTORICAL:
+      productType: EO:ECMWF:DAT:CEMS_FIRE_HISTORICAL_V1
+      providerProductType: reanalysis
+      variable:
+        - fire_weather_index
+      version:
+        - "4_1"
+      format: '"grib"'
```
Comment (amarandon) [inline eodag/resources/providers.yml:4834]: @jlahovnik this works but doesn't feel right

File: eodag/resources/providers.yml [hunk 2]
```diff
@@ -4812,6 +4824,34 @@
...
+        id: '$.id'
+        <<: *day_month_year
+        providerProductType:
+          - '{{"product_type": "{providerProductType}"}}'
+          - '$.null'
+        version:
+          - '{{"system_version": {version}}}'
+          - '$.null'
+        format:
+          - '{{"data_format": {format}}}'
```
Comment (jlahovnik) [inline eodag/resources/providers.yml:4847]: if you write '{{"data_format": "{format}"}}' here you should not need the double quotes above

Comment (amarandon) [reply eodag/resources/providers.yml]: Thanks! It works and looks much better. Just pushed the change.

**Comment by jlahovnik** [comment]
You can remove the note from the PR description.

File: eodag/resources/providers.yml [hunk 3]
```diff
@@ -2782,6 +2784,16 @@
               "hydrological_year": {completionTimeFromAscendingNode#get_hydrological_year}
             }}
           - '{$.completionTimeFromAscendingNode#get_hydrological_year}'
+    FIRE_HISTORICAL:
```
Comment (jlahovnik) [inline eodag/resources/providers.yml:2787]: Why did you re-add FIRE_HISTORICAL to cop_cds? It has been removed from there (and moved to cop_ewds).

File: eodag/resources/providers.yml [hunk 4]
```diff
@@ -4812,6 +4824,34 @@
...
         defaultGeometry: 'POLYGON((180 -90, 180 90, -180 90, -180 -90, 180 -90))'
         orderLink: 'https://gateway.prod.wekeo2.eu/hda-broker/api/v1/dataaccess/download?{{"location": "{downloadLink}","product_id":"{id}", "dataset_id": "EO:ECMWF:DAT:INSITU_GLACIERS_EXTENT"}}'
+    FIRE_HISTORICAL:
+      productType: EO:ECMWF:DAT:CEMS_FIRE_HISTORICAL_V1
+      providerProductType: reanalysis
+      variable:
+        - fire_weather_index
+      version:
+        - "4_1"
+      format: grib
```
Comment (jlahovnik) [inline eodag/resources/providers.yml:4834]: Please add output_extension: .grib to the download config, otherwise the files are stored without extension.

**Review by jlahovnik — CHANGES_REQUESTED** [review]

**Review by jlahovnik — APPROVED** [review]

**Review by sbrunato — APPROVED** [review]

**Pull request merged**